src/condor_scripts/condor_watch_q

```diff
@@ -85,6 +85,14 @@ def parse_args():
     parser.add_argument(
         "-clusters", nargs="+", metavar="CLUSTER_ID", help="Which cluster IDs to track."
     )
+    parser.add_argument(
+        "-since",
+        type=int,
+        nargs=1,
+        default=None,
+        metavar="CLUSTER_ID",
+        help="Track all clusters since specified cluster ID."
```

Comment:
I wonder if this should be "larger than the" instead of "since" in the text; it's more technically correct (although I don't expect cluster IDs being monotonically increasing with time to change soon) and maybe better because it doesn't repeat the word in its definition?